src/WCSimDetectorConfigs.cc

```diff
@@ -148,8 +148,8 @@ void WCSimDetectorConstruction::Cylinder_60x74_20inchBandL_14perCent()
   WCPMTName           = PMT->GetPMTName();
   WCPMTExposeHeight   = PMT->GetExposeHeight();
   WCPMTRadius         = PMT->GetRadius();
-  WCIDDiameter          = 74.0*m;
-  WCIDHeight            = 60.0*m;
+  WCIDDiameter          = 74*m;
```

Comment:
Can you include the .0 in WCIDDiameter and WCIDHeight for these two cylinder configurations? I want them to disappear from the diff.